.circleci/config.yml

```diff
@@ -32,12 +32,13 @@ jobs:
           command: |
             python3 -m venv venv
             . venv/bin/activate
-            pip install -r requirements.txt
+            sudo pip install pipenv
+            pipenv install
 
       - save_cache:
           paths:
             - ./venv
-          key: v1-dependencies-{{ checksum "requirements.txt" }}
+          key: v1-dependencies-{{ checksum "Pipfile.lock" }}
       
       - run:
           name: start gunicorn
```

Comment:
pipenv run gunicorn 'service.microservice:start_service()'

Reply:
same with tests.
pipenv run python -m pytest --cov=service tests/  --cov-fail-under=100
I couldn't get coveralls to work with pipenv.  Might need to run it as a compound command along with pytest.

Reply:
similar to above: 2ca1926 
re coveralls: used `pipenv run coveralls`